src/common/utils/utils.go

```diff
@@ -295,3 +295,11 @@ func FindNamedMatches(regex *regexp.Regexp, str string) map[string]string {
 	}
 	return results
 }
+
+// TrimProxyPrefix remove proxy prefix from original original project name
+func TrimProxyPrefix(projectName, repo string) string {
+	if strings.HasPrefix(repo, projectName+"/") {
```

Comment:
this `if` is not needed TrimPrefix will handle it if the string does not have the prefix.  

This func also should not be part of common package, it's proxy specific.